grpc/src/compiler/python_generator.cc

```diff
@@ -175,6 +150,20 @@ class StubGenerator : public BaseGenerator {
   }
 
  private:
+  bool SaveStub(const std::string& filename, const Imports& imports,
```

Comment:
Same here, if making these private is needed for the functionality, sure, but if its a drive-by "I am making the code better" you better be damn sure its near impossible nobody could be calling these.